### ISSUE TITLE
Transmission rechecks all torrents upon restart

### ISSUE THREAD

**Issue by kanyck** [transmission/transmission#2565]
transmission 3.0 set up in a FreeBSD jail. After reboot/jail restart it rechecks all the torrents...
Probably it's the same as #2434. No idea where to look...

**Comment by ckerr** [comment]
Can't be the same as #2434; that was a regression in the nightlies.

Transmission is looking for the check history in the files in the resume/ subdirectory underneath the config directory. Maybe that directory is getting cleared out by your system between sessions?

**Comment by kanyck** [comment]
No, it's full of `.resume` files. However some torrents are stopped now so `.resume` files for them are somewhat dated.
That was the first thing I checked when I saw it's rechecking over again.

**Comment by ckerr** [comment]
Should be fixed as a side-effect of #2626.
